### PR TITLE
chore: source updated from crowdin

### PR DESCRIPTION
**Pull request by yamilmedina**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Enable workflow to be run by translators, so we can update source (default: `strings.xml)` from crowdin

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: .github/workflows/crowdin-source-updater.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                    download_sources: true
+                    push_sources: true
+                    upload_sources: false
+                    download_translations: true
+                    upload_translations: false
+
+                    create_pull_request: true
+                    localization_branch_name: chore/sync-and-update-localization-source
+                    commit_message: "chore: update localization source from Crowdin"
+                    pull_request_title: "chore: update localization source from Crowdin"
```
Comment (gongracr) [inline .github/workflows/crowdin-source-updater.yml]:
```suggestion
                    pull_request_title: "chore: update localization sources from Crowdin"
```

File: .github/workflows/crowdin-source-updater.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+# **What it does**: Download new source from crowdin project.
+# **Why we have it**: We want to be able to pull-update base (EN) translations to the project.
+# **Who does it impact**: Everyone collaborating on the project, and the translators.
+# For more info: https://github.com/crowdin/github-action/blob/master/action.yml
+
+on:
+    push:
+        branches: [ "chore/crowdin-source-updater" ] # Delete after tst.
+    workflow_dispatch:
```
Comment (gongracr) [inline .github/workflows/crowdin-source-updater.yml:9]: Should we remove or is it needed for the GH action to run?

**Review by gongracr — APPROVED** [review]
Looking very bueno! 🍘

**Comment by AndroidBob** [comment]
Build [1189](https://10.10.124.134/job/AR-build-pipeline/1189/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7788006)(7788006)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7788006)(7788006)-staging-compat.apk)

File: .github/workflows/crowdin-source-updater.yml (hunk 2 above)
Comment (yamilmedina) [reply .github/workflows/crowdin-source-updater.yml:9]: Yes we need the `workflow_dispatch` so we can trigger it manually, I removed the push block tho, the test ran successfully

Comment (vitorhugods) [reply .github/workflows/crowdin-source-updater.yml:9]: > so we can trigger it manually

Who, exactly, and how? We, developers, by hitting the button in `Actions`, or we have some more stuff going on?

**Review by vitorhugods — APPROVED** [review]
LGTM, just small question

**Comment by yamilmedina** [comment]
> LGTM, just small question

It could be us, as developers, but we mainly do this by updating the code directly. The intended use case for this, would be 99% for Astrid if she wants to pull some translations made to the base `strings.xml` on crowdin without needing to do it by code or having a developer to do it =)

**Comment by AndroidBob** [comment]
Build [1196](https://10.10.124.134/job/AR-build-pipeline/1196/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7788668)(7788668)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7788668)(7788668)-staging-compat.apk)

**Pull request merged**